cost/aws/reserved_instances/recommendations/aws_reserved_instance_recommendations_with_purchase.pt

```diff
@@ -0,0 +1,355 @@
+name "AWS Reserved Instances Recommendations with Purchase"
+rs_pt_ver 20180301
+type "policy"
+short_description "A policy that sends email notifications when AWS RI Recommendations are identified. NOTE: These RI Purchase Recommendations are generated by AWS. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/reserved_instances/recommendations) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
Include that the user can purchase the RI after approval.

A policy that sends email notifications when AWS RI Recommendations are identified and allows the user the make RI purchase with approval.